code/scripting/doc_parser.cpp

```diff
@@ -202,8 +202,8 @@ class TypeVisitor : public BaseVisitor {
 
 	antlrcpp::Any visitMap_type(ArgumentListParser::Map_typeContext* context) override
 	{
-		const auto keyType = visit(context->type(0)).as<ade_type_info>();
-		const auto valueType = visit(context->type(1)).as<ade_type_info>();
+		auto keyType = visit(context->type(0)).as<ade_type_info>();
+		auto valueType = visit(context->type(1)).as<ade_type_info>();
```

Comment:
Why remove the `const` here?

Reply:
Uh ... let me reverse those and the other accidental const changes.